src/routes/jira/sync/jira-sync-post.ts

```diff
@@ -4,7 +4,7 @@ import { NextFunction, Request, Response } from "express";
 import { findOrStartSync } from "~/src/sync/sync-utils";
 
 export const JiraSyncPost = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
-	const { installationId: gitHubInstallationId, syncType } = req.body;
+	const { installationId: gitHubInstallationId, syncType, appId } = req.body;
```

Comment:
just a quick one, can you please do 
`	const { installationId: gitHubInstallationId, syncType, appId: gitHubAppId } = req.body;` instead? I think that's more readable. As `appId` has some meaning inside the `GitHubServerApp`.

Reply:
Updated to ` appId: gitHubAppId `